encoder/vaapiencoder_base.cpp

```diff
@@ -51,6 +51,7 @@ VaapiEncoderBase::VaapiEncoderBase():
     m_videoParamCommon.frameRate.frameRateNum = 30;
     m_videoParamCommon.frameRate.frameRateDenom = 1;
     m_videoParamCommon.intraPeriod = 15;
+    m_videoParamCommon.ipPeriod = 1;
```

Comment:
do we need change to this to another name since it means b freames between non bframe.